lib/xcode/install.rb

```diff
@@ -184,8 +264,9 @@ def install_dmg(dmg_path, suffix = '', switch = true, clean = true)
       FileUtils.rm_f(dmg_path) if clean
     end
 
-    def install_version(version, switch = true, clean = true, install = true, progress = true, url = nil, show_release_notes = true)
-      dmg_path = get_dmg(version, progress, url)
+    # rubocop:disable Metrics/ParameterLists
+    def install_version(version, switch = true, clean = true, install = true, progress = true, url = nil, show_release_notes = true, progress_block = nil)
```

Comment:
this `progress_block` always seems to be `nil`?

Reply:
Yeah, the `install_version` with `progress_block` would be called from fastlane.ci, I don't think we need it for anything on the `xcode-install` CLI, as we print the output in real-time using stdout